components/cloud/aws/src/metrics.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+use lazy_static::*;
+use prometheus::*;
+
+lazy_static! {
+    pub static ref S3_REQUEST_HISTOGRAM_VEC: HistogramVec = register_histogram_vec!(
+        "s3_request_duration_seconds",
+        "Bucketed histogram of s3 requests duration",
+        &["type"]
+    )
```

Comment:
1. looks strange that the metrics is s3-only. there could be a new label (schema/provider/whatever) with value "s3".
2. the metrics name should be namespaced like `tikv_cloud_…`

Reply:
Done, PTAL again~ @kennytm